README.md

```diff
@@ -4,8 +4,11 @@ An AWS Lambda microservice presenting MIPS chart of accounts data
 ## Architecture
 
 This microservice is designed to retrieve a chart of accounts from a third-party API and present the data in a useful format.
-The initial implementation formats all accounts into a JSON string,
-future iterations will add formats to replace existing static files and filter out inactive accounts.
+
+Formats available:
+* A dictionary mapping all active and inactive accounts to their friendly names.
+* A list of valid values for our `CostCenter` tag.
+* A list of valid values for either `CostCenter` or `CostCenterOther` tags.
```

Comment:
can you please explain what is the purpose of doing creating these two lists?  I think all we need is one list of all cost centers.

Reply:
i've found two different use cases for the data: the short list of tags is used to set [service catalog tag options restricting the value of CostCenter](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/a149c14b8513a8b159bb5228a251b907e3535e7a/sceptre/scipool/config/prod/sc-tag-options.yaml#L17) and the full list of all values is used for [verifying valid values for either CostCenter or CostCenterOther](https://github.com/Sage-Bionetworks/scicomp-provisioner/blob/79afb68321e47e9d9153aa82e61f613e9b220937/.pre-commit-config.yaml#L27-L28)

so I created two similar transforms to address these use cases in an attempt to move all of the related business logic into the lambda itself.